apps/examples/st_things/contents/sampleDevice.json

```diff
@@ -109,10 +109,10 @@
             "frequency": 1
         },
         "filePath":{
-            "svrdb": "/mnt/artikserversecured.dat",
-            "provisioning": "/mnt/provisioning.dat",
-            "certificate": "/rom/certificate",
-            "privateKey": "/rom/privatekey"
+            "svrdb": "artikserversecured.dat",
```

Comment:
Commit message is something wrong. (Not matched)

Reply:
Add commits aside